vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -121,14 +115,14 @@ ent jhipster entity Conference
 ent prj build
 ```
 
-Note: Using the `ent prj` wrapper avoids having to build each part of the project individually. The first run using `ent prj` can be slower due to MFE node downloads. See [this tutorial](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) for more details.
+> Note: Using the `ent prj` wrapper avoids having to build each part of the project individually. The first run using `ent prj` can be slower due to MFE node downloads. See [the BLueprint tutorial](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) for more details.
 
 ### Prepare and Publish a Bundle
-Use the publication system (pbs) to assemble your Entando project into a bundle that can be loaded into Kubernetes. You'll need your GitHub credentials, an empty GitHub repository to hold your bundle artifacts and a Docker Hub account or organization.
-1. Initialize the bundle directory
+Use the publication system (pbs) to assemble your Entando project into a bundle that can be loaded into Kubernetes. You'll need your GitHub credentials, an empty GitHub repository to hold your bundle artifacts, and a Docker Hub account or organization.
+1. Initialize the bundle directory. This method accepts SSH and Git URLs.
```

Comment:
"SSH and GIT URLs" -> "both HTTPS and SSH URLs".   
Technically we only include github URLs, for now - those are the two flavors github offers.

Reply:
@nathan this is wrong in the release notes, too!

Reply:
And I see you fixed it, thanks.